catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/security/ShareableMetacardAccessPluginTest.java

```diff
@@ -52,12 +56,16 @@ public void setUp() {
     subject = mock(Subject.class);
 
     accessPlugin =
-        new WorkspaceAccessPlugin() {
+        new ShareableMetacardAccessPlugin() {
           @Override
           protected Subject getSubject() {
             return subject;
           }
         };
+
+    ShareableMetacardImpl.setShareableMetacardType(
+        new MetacardTypeImpl(
+            "metacard.sharing", Arrays.asList(new SecurityAttributes(), new CoreAttributes())));
```

Comment:
❗️ Can probably remove since nobody uses the duplicated descriptor state in `ShareableMetacardImpl` but the descriptor state that lives in `MetacardImpl`.